### ISSUE TITLE
'Download photo' not working

### ISSUE THREAD

**Issue by Ngseri** [iionly/tidypics#50]
First off, thank you iionly for updating this plugin.

I downloaded the latest version of tidypics for Elgg 3.3.8 and tested the plugin. One issue I found is that 'Download photo' function open the photo to view instead of downloading the photo. 

To make it work, I changed the 'href' value of 'download' menu from 'photos/download/$photo_guid' to '$photo->getDownloadURL()' in mod/tidypics/views/default/resources/tidypics/photos/image/view.php. 

Thanks!

**Comment by iionly** [comment]
Thank for the notification. I had noticed that something seems wrong with the download link while testing but then forgot...

I've fixed it slightly differently because your fix would only work when clicking on the Download button on the views page. But if someone adds a link somewhere else it would not work. Therefore, I fixed it in the download view file directly. I'll make a PR soon. I don't know if I will make a new release just with this one issue fixed so soon after the initial release for Elgg 3. I might wait a bit longer just to see if any other bugs show up.
